examples/mugen/retrieval/model.py

```diff
@@ -4,30 +4,37 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Any
+import warnings
+from typing import Any, Tuple
+
+import torch
 
 from examples.mugen.retrieval.video_clip import videoclip
 from pytorch_lightning import LightningModule
+from torchmetrics import Recall
+
 from torchmultimodal.modules.losses.contrastive_loss_with_temperature import (
     ContrastiveLossWithTemperature,
 )
 
 
 class VideoCLIPLightningModule(LightningModule):
     """PyTorch Lightning module for evaluating VideoCLIP model.
-
     Args:
         logit_scale (float): Initial log-temperature value for contrastive loss funtion.
-            Defaults to ``3.8282``, MUGEN's log-temperature value after training.
+            Defaults to ``0.07``, MUGEN's log-temperature value at initialization.
```

Comment:
Where did the ``3.8282`` come from previously? Why "after training" --> "at initialization"?

Reply:
3.8282 came from the checkpoint as the trained temperature parameter, but mugen initialized temperature to 0.07 so i think it makes more sense for the lightning module to initialize to 0.07 and only use 3.8282 when using the checkpoint